dm/docs/RFCS/20220124_enhance_task_manageability.md

```diff
@@ -0,0 +1,127 @@
+| delete   | `dmctl source delete --source-name="source1"`                           | --source-name                | delete a source that has no running tasks.                          |
+| get      | `dmctl source get --source-name="source1" --output="source1.yaml"`      | --source-name, --output      | show the source config in yaml format, also support output to file. |
+| list     | `dmctl source list`                                                     |                              | list all source in current cluster.                                 |
+| status   | `dmctl source status --source-name="source1"`                           | --source-name                | show source detail status.                                          |
+| enable   | `dmctl source enable --source-name="source1"`                           | --source-name                | enable a disabled source.                                           |
+| disable  | `dmctl source disable --source-name="source1"`                          | --source-name                | disable a source and also stop the running subtasks of this source. |
+| transfer | `dmctl source transfer --source-name="source1" --worker-name="worker1"` | --source-name, --worker-name | transfers a source to a free worker.                                |
+
+### DMCTL commands for Relay
+
+| command | Full Syntax Example                                                                                                                     | Flags                                 | Description                                                                  |
+| ------- | --------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------- | ---------------------------------------------------------------------------- |
+| start   | `dmctl relay start --source-name="source1.yaml" --worker-name="worker1"`                                                                | --source-name, --worker-name          | start relay for a source on a worker.                                        |
+| stop    | `dmctl relay stop --source-name="source1.yaml" --worker-name="worker1"`                                                                 | --source-name, --worker-name          | stop relay for a source on a worker.                                         |
+| pause   | `dmctl relay pause --source-name="source1.yaml" --worker-name="worker1"`                                                                | --source-name, --worker-name          | pause relay for a source on a worker.                                        |
+| resume  | `dmctl relay resume --source-name="source1.yaml" --worker-name="worker1"`                                                               | --source-name, --worker-name          | resume relay for a source on a worker.                                       |
```

Comment:
Is pause/resume necessary?

Reply:
i thinks not, removed in https://github.com/pingcap/tiflow/pull/4458/commits/599193432235e57590bfd0837d6425ceda237674